vert.x/community/configmap/booster.yaml

```diff
@@ -10,11 +10,11 @@ environment:
   staging:
     source:
       git:
-        ref: v25
+        ref: v26
   production:
     source:
       git:
-        ref: v25
+        ref: v26
     metadata:
```

Comment:
Now is a good time to remove this metadata from these boosters (they are already defined in metadata.yaml)